test/deploy_test.rb

```diff
@@ -32,7 +32,7 @@ def check_process(pid)
   end
 
   describe "setup" do
-    before { mina "setup" }
+    before { system 'echo ---; cat /etc/issue; netstat -nlpt; which systemctl; echo ---'; mina "setup" }
```

Comment:
this seems like extra debugging

Reply:
I'm trying to figure out why the test fails

Reply:
@devvmh And I probably need your help here. Any idea what might be wrong? Any way to ssh to the machine? I've got a couple of ideas, like `cat` `~/.ssh`, `/etc/ssh`, but I hope you've got better suggestions...